doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -0,0 +1,255 @@
+==========================
+Dam break VOF
+==========================
+
+This example simulates the dam break experiments of `Martin and Moyce (1952)`_. 
+
+.. _Martin and Moyce (1952): https://royalsocietypublishing.org/doi/abs/10.1098/rsta.1952.0006
+
+A liquid is fixed behind a dam at the left most corner of
+a rectangular domain as shown in the figure below.
+At :math:`t = 0` seconds, the dam is removed, and 
+the liquid is released into the total simulation domain. 
+The corresponding parameter file is 
+``gls_VOF_dam-break_Martin_and_Moyce.prm``.
```

Comment:
Add the location of the example prm file. Look at the 2D Lid-Driven Cavity Flow example (location of the example).

Reply:
I don't think that that is necessary. At this point anyone who knows how to move around the documentation tree knows how to get to an example